app/views/ctc/portal/portal/edit_info.html.erb

```diff
@@ -9,7 +9,7 @@
       <%= link_to t("general.back"), :back %>
     </div>
     <div class="question-wrapper">
-      <%= form_with model: @form, url: ctc_portal_resubmit_path, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
+      <%= form_with url: ctc_portal_resubmit_path, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
```

Comment:
It looks like `@form` was nil, so `model: @form` was doing nothing. Thanks for deleting it!